src/components/Layout.module.scss

```diff
@@ -1,16 +1,46 @@
 @import 'src/styles/colors';
 
 .Layout {
+  width: 650px;
+  background: $white-color;
+  margin: 60px 0;
 }
 
 .topBar {
-    height: 30px;
-    font-size: 20px;
-    padding: 10px;
-    background: $background-color;
+    background: $background-color2;
+    // padding: 3px 6px;
+    display: flex;
+    justify-content: space-between;
+
+    & button {
+     
+    }
+}
+.topBarContainer {
+  padding: 3px 6px;
+}
+.topBarHeadline {
+  font-size: 10px;
+  margin: 0;
+  color: $headline-color;
+}
+.topBarStationsName {
+  font-size: 18px;
+  margin: 0;
+  font-weight: 500;
+  padding-left: 12px;
```

Comment:
If it needs so many sub-styles, maybe it would be a good idea to create a separate component just for that?